Misc/NEWS.d/next/Library/2019-11-17-17-32-35.bpo-38615.OVyaNX.rst

```diff
@@ -0,0 +1,2 @@
+The :class:`~imaplib.IMAP4` and :class:`~imaplib.IMAP4_SSL` now has optional
```

Comment:
Again "The classes ...", and "have an optional timeout parameter for their constructors."